reconcile/slack_sender.py

```diff
@@ -0,0 +1,77 @@
+import sys
+import logging
+
+import reconcile.queries as queries
+from utils.slack_api import SlackApi
+
+from utils.state import State
+
+QONTRACT_INTEGRATION = 'slack-sender'
+
+
+def collect_to(to):
```

Comment:
i think we should re-use the existing collect_to function and not include slack channels in it (will also be required due to the previous comment)